packages/react-docs/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@0xproject/react-docs",
+  "version": "0.0.1",
+  "description": "React documentation component for rendering TypeDoc & Doxity generated JSON",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "lint": "tslint --project . 'src/ts/**/*.ts' 'src/ts/**/*.tsx'",
+    "build": "tsc",
+    "build:example": "NODE_ENV=production webpack; exit 0;",
```

Comment:
do we want to hide if webpacks to fail? `;` sucks on windows so trying to avoid these

Reply:
Luckily we don't need it in this repo 💃 Removed.